FLEDGE.md

```diff
@@ -44,7 +44,7 @@ See [the Protected Audience API specification](https://wicg.github.io/turtledove
     - [5.4 Reporting IDs](#54-reporting-ids)
     - [5.5 Losing Bidder Reporting](#55-losing-bidder-reporting)
   - [6. Additional Bids](#6-additional-bids)
-    - [6.1 Auction Nonce](#61-auction-nonce)
+    - [6.1 Auction and Bid Nonces](#61-auction-nonce)
```

Comment:
This doesn't actually change the heading name.  I'd suggest either reverting this line or changing the heading name and all the links to it.

Reply:
My mistake - I had that change in my draft, but forgot to make it here on GitHub. Fixed. Thanks.